src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(a) {
+    count++;
+
+    return function(b) {
+      if (count % 2 === 0 && count > 3) {
+        return 'Bzzz... Error!';
+      } else {
+        return a + b;
+      };
```

Comment:
just use return without else - it will surely be reached when error won't appear